chart/values.yaml

```diff
@@ -0,0 +1,191 @@
+# Default values for policy-bot.
+# This is a YAML-formatted file.
+
+# -- Number of replicas to deploy
+replicaCount: 1
+
+image:
+  # -- Docker image repository
+  repository: palantirtechnologies/policy-bot
+  # -- Image pull policy
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion
+  tag: ""
+
+# -- Image pull secrets for private registries
+imagePullSecrets: []
+
+# -- Override the name of the chart
+nameOverride: ""
+
+# -- Override the full name of the chart
+fullnameOverride: ""
+
+# -- Pod annotations
+podAnnotations: {}
+
+# -- Pod security context
+podSecurityContext: {}
+
+# -- Container security context
+securityContext:
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+
+service:
+  # -- Service type
+  type: ClusterIP
+  # -- Service port
+  port: 8080
+  # -- Service annotations
+  annotations: {}
+
+ingress:
+  # -- Enable ingress
+  enabled: false
+  # -- Ingress class name
+  className: ""
+  # -- Ingress annotations (use for cert-manager: cert-manager.io/cluster-issuer: "letsencrypt-prod")
+  annotations: {}
+  # -- Ingress hosts configuration
+  hosts:
+    - host: policy-bot.local
+      paths:
+        - path: /
+          pathType: Prefix
+  # TLS configuration (3 options):
+  # Option 1: Provide cert/key directly, chart creates the secret
+  # Option 2: Reference an existing secret via existingSecret
+  # Option 3: Use cert-manager by adding annotation above and leave false
+  tls:
+    # -- Enable TLS
+    enabled: false
+    # -- Base64 encoded TLS certificate (Option 1)
+    certificate: ""
+    # -- Base64 encoded TLS private key (Option 1)
+    privateKey: ""
+    # -- Name of existing TLS secret (Option 2)
+    existingSecret: ""
+    # -- Hostname for TLS (defaults to first ingress host)
+    host: ""
+
+# -- Resource requests and limits
+resources: {}
+  # limits:
+  #   cpu: 500m
+  #   memory: 256Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+# -- Node selector for pod assignment
+nodeSelector: {}
+
+# -- Tolerations for pod assignment
+tolerations: []
+
+# -- Affinity rules for pod assignment
+affinity: {}
+
+# Policy-bot specific configuration
+config:
```

Comment:
Can we have an "existingConfigMap" or similar option? There are likely to be cases where people want to set options that aren't in the config, which is something I've done in the past for other helm charts that support it. 

[Here](https://github.com/grafana/alloy/blob/e2e96e95ff0dab600befbe63165e10eea096b968/operations/helm/charts/alloy/values.yaml#L31-L41) is a pretty standard example of how others set configs.